Testing/Exec/ReactEval_C/main.cpp

```diff
@@ -402,10 +402,13 @@ main (int   argc,
                       dt_incr, time,
                       ode_iE, amrex::Gpu::gpuStream());
 #else
-                fc_tmp_lcl = react(tmp_vect, tmp_src_vect,
-                                   tmp_vect_energy, tmp_src_vect_energy,
-                                   &dt_incr, &time,
-                                   &ode_iE, &ncells, amrex::Gpu::gpuStream());
+                react_1(box,
+                      rhoY, frcExt, T,
+                      rhoE, frcEExt,
+                      fc, mask,
+                      dt_incr, time);
+#endif
+
```

Comment:
@drummerdoc If you look at the current diff in GitHub, these lines haven't changed now. I think it was showing as changed because of a conflicted git merge which I fixed now by rebasing (hency why there are only 2 commits in this PR now).